src/terraform/terraform_install.md

```diff
@@ -1,9 +1,48 @@
-![ ](./img//terraform_install.png)
+<h1> Installing Terraform</h1>
 
-### Terraform Install
+<h2> Table of Contents </h2>
 
-Please note that for now our [Terraform plugin](https://github.com/threefoldtech/terraform-provider-grid) is supported on Linux and MacOS. Windows
+- [Introduction](#introduction)
+- [Install Terraform](#install-terraform)
+  - [Install Terraform on MAC and Linux](#install-terraform-on-mac-and-linux)
+  - [Install Terraform on Windows](#install-terraform-on-windows)
+- [ThreeFold Terraform Plugin](#threefold-terraform-plugin)
+- [Questions and Feedback](#questions-and-feedback)
 
-Get Terraform from the [download page](https://www.terraform.io/downloads.html) on the official site, or install using you system's package manager. please check the [installation manual](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+***
+
+## Introduction
+
+There are many ways to install Terraform depending on your operating system. Terraform is available for Linux, MAC and Windows.
+
+## Install Terraform
+
+You can get Terraform from the Terraform website [download page](https://www.terraform.io/downloads.html). You can also install it using your system package manager. The Terraform [installation manual](https://learn.hashicorp.com/tutorials/terraform/install-cli) contains the essential information for a proper installation.
+
+We cover here the basic steps for Linux, MAC and Windows for convenience. Refer to the official Terraform documentation if needed.
+
+### Install Terraform on MAC and Linux
+
+To install Terraform on MAC and Linux, a quick way is to first install Brew and then install Terraform.
```

Comment:
Brew isn't the only way to install Terraform on Linux; it's not even the official way to install it from the [site itself](https://developer.hashicorp.com/terraform/downloads).
![image](https://github.com/threefoldtech/info_grid/assets/109473873/471d8179-ab5e-4612-92a3-ca9c1337e146)
 
Can you add it here and in src/computer_it_basics/cli_scripts_basics.md too?

Reply:
Of course. Let me know if it's OK now.